meilisearch/index.py

```diff
@@ -1168,6 +1152,67 @@ def reset_sortable_attributes(self) -> Dict[str, int]:
             self.__settings_url_for(self.config.paths.sortable_attributes),
         )
 
+    # TYPO TOLERANCE SUB-ROUTES
```

Comment:
Does it seem a responsibility for a `settings.py` file, don't you think?

Reply:
@brunoocasali Yes, I thought the same thing about adding another `settings` in the `index.py` file but isn't it better to do it during the rest of maintenance in a separate PR? Or do you prefer I modify it in this PR?

Reply:
I don't see a problem if you track that change, like, add the new configurations in a new file, and then create an index to address the following changes in the new place.